org.opent2t.sample.thermostat.superpopular/test/thermostatTests.js

```diff
@@ -100,7 +100,7 @@ function runThermostatTests(settings) {
     test.serial('SetTargetTemperatureHigh', t => {
         setData(t);
         return OpenT2T.invokeMethodAsync(translator, SchemaName, 'getDevicesTargetTemperatureHigh', [deviceId]).then((initialTemperature) => {
-            return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesTargetTemperatureHigh', [deviceId, {'temperature': 22}]).then(() => {
+            return OpenT2T.invokeMethodAsync(translator, SchemaName, 'postDevicesTargetTemperatureHigh', [deviceId, { 'temperature': 22, 'units': 'c' }]).then(() => {
```

Comment:
Not really, I just needed the "units" object for Nest. I think the spaces were added by VS automatically. Should I remove them?

---
In reply to: [98082435](https://github.com/openT2T/translators/pull/150#discussion_r98082435) [](ancestors = 98082435)